vendor/modules.txt

```diff
@@ -1044,8 +1044,8 @@ kubevirt.io/api/export
 kubevirt.io/api/export/v1alpha1
 kubevirt.io/api/instancetype
 kubevirt.io/api/instancetype/v1beta1
-# kubevirt.io/containerized-data-importer-api v1.57.0-alpha1
-## explicit; go 1.19
+# kubevirt.io/containerized-data-importer-api v1.59.0
+## explicit; go 1.21
```

Comment:
this means that we have to upgrade to RHEL 9.4 when releasing this fix, right?

Reply:
we already did upgrade to 9.4. what is the concern?

Reply:
ok, so if we create a new branch for 2.6.2, we need to keep in mind that we should change the base image to RHEL 9.4 there as well then